Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -4611,5 +4611,41 @@
         "Length": 14
       }
     ]
+  },
+  {
+    "Input": "semana pasada",
```

Comment:
"semana" is being wrongly recognized as an Age entity. Can you add a spec for that and look into the cause of the issue? This PR should include that fix too. Thanks.